CRM/Sepa/BAO/SEPAMandate.php

```diff
@@ -48,9 +50,14 @@ static function add(&$params) {
         // new mandate, use the default creditor
         $default_creditor = CRM_Sepa_Logic_Settings::defaultCreditor();
         $params['creditor_id'] = $default_creditor->id;
-      } else {
+      }
+      else {
         // existing mandate, get creditor
-        $params['creditor_id'] = civicrm_api3('SepaMandate', 'getvalue', array ('id' => $params['id'], 'return' => 'creditor_id'));
+        $params['creditor_id'] = \Civi\Api4\SepaMandate::get(TRUE)
```

Comment:
I thought about that, too, but find it too important not to be left out, so that it's clear we really want to check permissions and haven't just forgot to pass `FALSE`.

Reply:
Though I've spotted some calls where no argument is given...